modules/cos-coredns/assets/cloud-config.yaml

```diff
@@ -1,5 +1,3 @@
-#cloud-config
```

Comment:
@ludoo is there significance to this comment or can it be removed?

Reply:
it's actually needed. Linter should not break (license is still valid if it does not start on the first line afaik).

Reply:
removed

Reply:
Comments restored. Also excluded `cloud-config.yaml`-files from [check_headers func](https://github.com/terraform-google-modules/terraform-google-container-vm/pull/43/files#diff-9aca4f6ccbe56e14f9edf295b7db1d3f)